packages/server/graphql/mutations/toggleTeamDrawer.ts

```diff
@@ -0,0 +1,65 @@
+import {GraphQLID, GraphQLNonNull} from 'graphql'
+import getRethink from '../../database/rethinkDriver'
+import {getUserId, isTeamMember} from '../../utils/authorization'
+import ToggleTeamDrawerPayload from '../types/ToggleTeamDrawerPayload'
+import {GQLContext} from '../graphql'
+import standardError from '../../utils/standardError'
+import TeamDrawerEnum from '../types/TeamDrawerEnum'
+import {DrawerTypes} from 'parabol-client/types/constEnums'
+
+const toggleTeamDrawer = {
+  type: GraphQLNonNull(ToggleTeamDrawerPayload),
+  description: `Show/hide the drawer in the team dashboard`,
```

Comment:
In this issue & PR, we've used the terms sidebar, drawer, and side sheet. I'd like to be consistent with the language so I've used drawer as it seems (according to [this](https://stackoverflow.com/questions/61074330/what-is-the-difference-between-matsidenav-and-the-matdrawer#:~:text=The%20difference%20between%20the%20two,the%20user%20to%20filter%20files) and [this](https://material.io/components/navigation-drawer)) that sidebars are global, e.g. for navigation, and drawers are more component-specific. Curious if you have any opinions on this!

Reply:
drawer is definitely what we should be calling it, especially because you can pull it out like a drawer.

we called the leftnav a sidebar because it used to always be on the side & you couldn't close it like a drawer, but now it definitely is one, but renaming a bunch of components doesn't sound like fun